### PR TITLE
Run clang-format on all .cc and .h files after rebasing.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

File: wally_the_cartographer/cmd/root.go [hunk 1]
```diff
@@ -398,6 +413,27 @@ func ensureAllReposAreCloned(managedRepositories []string, datadir string) error
...
+	fmt.Fprintf(logWriter, "=> Running clang-format on all *.cc and *.h files [%s]\n", workdir)
+	return filepath.Walk(workdir, func(rootPath string, info os.FileInfo, err error) error {
+		if info.IsDir() {
+			return nil
+		}
+		ext := filepath.Ext(rootPath)
+		if ext == ".cc" || ext == ".h" {
+			relPath, err := filepath.Rel(workdir, rootPath)
+			if err != nil {
+				log.Panicf("Could not make %s relative to %s. This should be impossible in this context.", rootPath, workdir)
```
Comment (cschuet) [inline wally_the_cartographer/cmd/root.go]: line too long?

**Review by cschuet — APPROVED** [review]

File: wally_the_cartographer/cmd/root.go (hunk 1 above)
Comment (SirVer) [reply wally_the_cartographer/cmd/root.go]: According to which metric? Fine, I introduced a few line breaks.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**